src/_ToggleSynapseObjects.scss

```diff
@@ -0,0 +1,16 @@
+// Import the theme css
+@use 'variables' as Portal;
+
+.ToggleSynapseObjects {
+  margin-top: 30px;
+  .reactSwitchContainer{
+    width: 60px;
+    position: absolute;
+    right: 0px;
+    margin-top: -54px;
```

Comment:
Lock it to the right side via right: 0px, and move it up via a negative margin-top.